.github/workflows/security.yml

```diff
@@ -0,0 +1,69 @@
+name: Security-check
+
+on:
+  pull_request: # change to push before merging code
+    branches:
+      - dev
```

Comment:
we don't have a dev branch we do PR on main.

Reply:
Ah, I missed that, thanks